tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -0,0 +1,712 @@
+		FinalZoneSizing( CurZoneEqNum ).ZoneHumRatAtCoolPeak = 0.01117049470250416; // AHRI condition at 80 F db / 67 F wb
+		Node( VRF( VRFCond ).CondenserNodeNum ).Temp = 35.0; // AHRI condition at 95 F db
+		Node( VRFTU( VRFTUNum ).VRFTUOAMixerOANodeNum ).Temp = 35.0; // AHRI condition at 95 F db
+		Node( VRFTU( VRFTUNum ).VRFTUOAMixerOANodeNum ).HumRat = 0.01; // don't care
+		FinalZoneSizing( CurZoneEqNum ).CoolDDNum = 1;
+		FinalZoneSizing( CurZoneEqNum ).TimeStepNumAtCoolMax = 1;
+		DesDayWeath.allocate( 1 );
+		DesDayWeath( 1 ).Temp.allocate( 1 );
+		DesDayWeath( FinalZoneSizing( CurZoneEqNum ).CoolDDNum ).Temp( FinalZoneSizing( CurZoneEqNum ).TimeStepNumAtCoolMax ) = 35.0;
+
+		FinalZoneSizing( CurZoneEqNum ).CoolDesTemp = 13.1; // 55.58 F
+		FinalZoneSizing( CurZoneEqNum ).CoolDesHumRat = 0.009297628698818194; // humrat at 12.77777 C db / 12.6 C wb
+
+		SimulateVRF( VRFTU( VRFTUNum ).Name, CurZoneNum, FirstHVACIteration, SysOutputProvided, LatOutputProvided, ZoneEquipList( CurZoneEqNum ).EquipIndex( EquipPtr ) );
+
+		ASSERT_EQ( 1, NumVRFCond );
+		ASSERT_EQ( ZoneInletAirNode, ZoneEquipConfig( VRFTU( VRFTUNum ).ZoneNum ).InletNode( 1 ) ); // only 1 inlet node specified above in ZoneHVAC:EquipmentConnections
+		ASSERT_EQ( 1.0, VRF( VRFCond ).CoolingCombinationRatio );
+		ASSERT_EQ( 10749.071979211991, VRF( VRFCond ).CoolingCapacity );
+		ASSERT_EQ( 10749.071979211991, VRF( VRFCond ).HeatingCapacity );
+		EXPECT_EQ( 0.0, VRF( VRFCond ).DefrostPower ); 
+
+		// test defrost operation Issue #4950 - Reverse cycle with timed defrost = 0
+
+		// set OA node temperatures for heating where defrost should be active
+		Node( VRF( VRFCond ).CondenserNodeNum ).Temp = VRF( VRFCond ).MaxOATDefrost - 1.0;
+		Node( VRF( VRFCond ).CondenserNodeNum ).HumRat = 0.005;
+		Node( VRFTU( VRFTUNum ).VRFTUOAMixerOANodeNum ).Temp = Node( VRF( VRFCond ).CondenserNodeNum ).Temp;
+
+		// set zone load to heating
+		ZoneSysEnergyDemand( CurZoneNum ).RemainingOutputRequired = VRF( VRFCond ).HeatingCapacity; // set load equal to the VRF heating capacity
+		ZoneSysEnergyDemand( CurZoneNum ).RemainingOutputReqToCoolSP = VRF( VRFCond ).HeatingCapacity + 1000.0; // simulates a dual Tstat with load to cooling SP > load to heating SP
+		ZoneSysEnergyDemand( CurZoneNum ).RemainingOutputReqToHeatSP = VRF( VRFCond ).HeatingCapacity;
+
+
+		SimulateVRF( VRFTU( VRFTUNum ).Name, CurZoneNum, FirstHVACIteration, SysOutputProvided, LatOutputProvided, ZoneEquipList( CurZoneEqNum ).EquipIndex( EquipPtr ) );
+
+		ASSERT_TRUE( VRF( VRFCond ).DefrostPower > 0.0 ); // defrost power should be greater than 0
+		DefrostWatts = VRF( VRFCond ).VRFCondRTF * ( VRF( VRFCond ).HeatingCapacity / 1.01667 ) * VRF( VRFCond ).DefrostFraction;
+		ASSERT_EQ( DefrostWatts, VRF( VRFCond ).DefrostPower ); // defrost power calculation check
+
+		// clean up
+		StdRhoAir = 0.0;
+		ZoneEqSizing.deallocate();
+		FinalZoneSizing.deallocate();
+		ZoneSysEnergyDemand.deallocate();
+		DesDayWeath.deallocate();
+
+	}
+}
```

Comment:
@rraustad Note this needs a new line at the end of HVACVariableRefrigerantFlow.unit.cc